lib/ansible/utils/module_docs_fragments/ldap.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2016, Peter Sagerson <psagers@ignorare.net>
+# Copyright: (c) 2016, Jiri Tyr <jiri.tyr@gmail.com>
+# Copyright: (c) 2017-2018 Keller Fuchs (@kellerfuchs) <kellerfuchs@hashbang.sh>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+
+class ModuleDocFragment(object):
+    # Standard LDAP documentation fragment
+    DOCUMENTATION = r'''
+options:
+  bind_dn:
+    description:
+    - A DN to bind with. If this is omitted, we'll try a SASL bind with the EXTERNAL mechanism.
```

Comment:
You should indent all the lists in this file by two more spaces to keep it the same like in all other files.